pkg/storage/storage_test.go

```diff
@@ -0,0 +1,225 @@
+package storage_test
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-psap/special-resource-operator/pkg/clients"
+	"github.com/openshift-psap/special-resource-operator/pkg/storage"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+const (
+	namespaceName = "test-ns"
+	resourceName  = "test-resource"
+)
+
+var nsn = types.NamespacedName{Namespace: namespaceName, Name: resourceName}
+
+func TestStorage(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Storage Suite")
+}
+
+var _ = Describe("Storage", func() {
```

Comment:
What do you think about Describe per tested function? This file tests only storage.go, so we don't actually need to state it in a specific Describe

Reply:
This allows me to have a common `BeforeEach` for multiple `Context`. How could I do it without the top-level `Describe`?